### DISCUSSION TITLE
Going lower than 6A?

### DISCUSSION THREAD

**Discussion by zulufoxtrot** [dingo35/SmartEVSE-3.5#157] [General]
What is the rationale for setting 6A as the lower charge limit?

Is it theoretically possible to go lower?

**Comment by dingo35** [comment]
The charging protocol defines the lowest charging current is 6A per phase. If you compile your firmware yourself you can lower that limit at your own risk; see docs, "Building the firmware".

**Comment by dingo35** [comment]
https://github.com/SmartEVSE/SmartEVSE-3/issues/44

**Comment by zulufoxtrot** [comment]
Thanks. I compiled the firmware with the special flag and it works.

I see that you mentioned 5 amps as the absolute lowest in the documentation, but I decided to try 4 amps (I like to live dangerously).

If I set "Min current" to 4 amps, I get weird behavior. The EVSE refuses to start charging (contactor stays open) and remains stuck at a 4A current target even if I try to override it or reboot the EVSE. The state is stuck at "connected to EV".

I wonder if it is caused by the car tripping, or some unforseen issue in the firmware?

**Comment by dingo35** [comment]
You are on uncharted territories here, so I don't know....

**Comment by zulufoxtrot** [comment]
OK thanks, appreciate the help 👍
